packages/ag-charts-enterprise/src/features/annotations/utils/styles.ts

```diff
@@ -58,8 +58,10 @@ export function setColor(
                 datum.fill = color;
                 datum.fillOpacity = opacity;
             } else {
-                if ('stroke' in datum) datum.stroke = color;
                 if ('strokeOpacity' in datum) datum.strokeOpacity = opacity;
+                if ('strokes' in datum) {
+                    datum.strokes = [color];
+                } else if ('stroke' in datum) datum.stroke = color;
```

Comment:
Nit: Can you put the assignment in it's own block, it's difficult to spot here?

Reply:
Fixed in https://github.com/ag-grid/ag-charts/commit/5a24596822df48d226e5c90bd8baab38723ede4f